src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/hooks/useEvent.js

```diff
@@ -9,26 +9,22 @@ export const useEvent = (eventId: string) => {
         enrollment: undefined,
         trackedEntity: undefined,
     });
-    const { data, error, loading } = useDataQuery(
+    const { data, error, loading, refetch } = useDataQuery(
         useMemo(
```

Comment:
Nitpick: Might as well drop useMemo and just move the query object up above useEvent.